### DISCUSSION TITLE
randomTrips default distribution

### DISCUSSION THREAD

**Discussion by Kunal-Kumar-Sahoo** [LucasAlegre/sumo-rl#204] [Q&A]
Hey @LucasAlegre, could you tell me what is the default distribution of traffic in the randomTrips.py script? Is there any way we can add other distributions in that script?

**Comment by LucasAlegre** [comment]
Hi @Kunal-Kumar-Sahoo, randomTrips.py is a script from SUMO, not from SUMO-RL. You can find their documentation at: https://sumo.dlr.de/docs/Tools/Trip.html
